lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/CLI/Server.hs

```diff
@@ -131,6 +131,51 @@ spec = do
                 logged `shouldNotContain` "TRACE"
                 logged `shouldNotContain` "DEBUG"
 
+        it "LOGGING - Serve debug logs for one component" $ \ctx -> do
+            withTempFile $ \logs hLogs -> do
+                let cmd = Command
+                        (commandName @t)
+                        ["serve"
+                        , "--node-port", show (ctx ^. typed @(Port "node"))
+                        , "--random-port"
+                        , "--genesis-block-hash", block0H
+                        , "--log-level", "debug"
```

Comment:
this line is not needed according to defaults - that is my expectation. this is nit-picking, but without it we would show that defaults work properly. Maybe it is good idea (in some next PR) to extend this test in such a way that we play with different combinations of general-log level and component log level and see that defaults are as expected. What happens when component log level is Info and general is Debug, or opposite? Also check two component logs with the same, different log levels...

Reply:
I needed to change the default global minimum severity back to Info to prevent debug-level MainLog messages from leaking out.

If the global log level is Info then there will be no Debug-level messages whatsoever. There is a check for that.
It the global log level is Debug then there will be Debug, Info, and higher-severity messages.
I added an assertion for showing that when Info is set then we will have for sure Info related logs.
I think that should cover it.

Reply:
@rvl would you mind adding this very comment as a note in place where we set this default to DEBUG ? This is good to know and have in the source code.

Reply:
I have put that change into #1282. There are are few extra debug severity MainLog messages printed due to this, but I think it's ok.